### PR TITLE
add FetchBotInfoResponse and make the message card could use no args constructor method in json serilizerbale

### PR DESCRIPTION
**Pull request by gwdgithubnom**
- To make more easy to set usrid for the batch message destination.
   - [x]  add method to set userId by List<User> in BatchMessageDestination.java 
- Add Rot info fetch operation
   - [x] add method to get InstanceContext and from LarkClient Instance(InstanceContext)
   - [x] add doFetchBotInfo method to get rob infomation
   - [x] add FetchBotInfoResponse.java and BotInfo.java
- Make the card entity could be serializable
   - [x] CardMessage.java ,ImageMessage and so on add a none args construct method to build the instance auto.

### CODE REVIEW

File: appframework-sdk/src/main/java/com/larksuite/appframework/sdk/core/protocol/OpenApiClient.java [hunk 1]
```diff
@@ -40,7 +40,10 @@
     private static final String SendMessageBatchPath = "/open-apis/message/v4/batch_send/";
     private static final String UploadImagePath = "/open-apis/image/v4/put/";
     private static final String FetchImagePath = "/open-apis/image/v4/get";
-
+    /**
+     * rot info
+     */
+    private static final String FetchROTInfoPath = "/open-apis/bot/v3/info/";
```
[the remark this answers is not in the document]
Comment (gwdgithubnom) [reply appframework-sdk/src/main/java/com/larksuite/appframework/sdk/core/protocol/OpenApiClient.java:46]: make the attribute about the robot information(include bot avatarUrl and name)

[the remark this answers is not in the document]
Comment (gwdgithubnom) [reply appframework-sdk/src/main/java/com/larksuite/appframework/sdk/core/protocol/OpenApiClient.java:46]: > whats the meaninig of ROT?

the response entity get from the api url: **/open-apis/bot/v3/info/**

[the remark this answers is not in the document]
Comment (xionghan00) [reply appframework-sdk/src/main/java/com/larksuite/appframework/sdk/core/protocol/OpenApiClient.java:46]: i think "bot" is a better name for "rot", could you help to change it

[the remark this answers is not in the document]
Comment (gwdgithubnom) [reply appframework-sdk/src/main/java/com/larksuite/appframework/sdk/core/protocol/OpenApiClient.java:46]: yeah, **    private static final String FetchROTInfoPath = "/open-apis/bot/v3/info/";**, this api path is in the larkapp serivce, and we see the api path contain the "bot", so call the rot name to "bot"

[the remark this answers is not in the document]
Comment (gwdgithubnom) [reply appframework-sdk/src/main/java/com/larksuite/appframework/sdk/core/protocol/OpenApiClient.java:46]: > i think "bot" is a better name for "rot", could you help to change it

should we change the name to rot? or keep with the api path same?

[the remark this answers is not in the document]
Comment (xionghan00) [reply appframework-sdk/src/main/java/com/larksuite/appframework/sdk/core/protocol/OpenApiClient.java:46]: my idea is    FetchROTInfoPath  =>    FetchBotInfoPath    ,    class Rot  =>  class BotInfo

**Pull request merged**